### PR TITLE
bias + peter-panning FIX

### PR DESCRIPTION
**Pull request by jocke1995**

### CODE REVIEW

File: StortSpelProjekt/Engine/src/Renderer/HLSL/SkyboxPixel.hlsl [hunk 1]
```diff
@@ -13,5 +13,5 @@ SamplerState point_Wrap : register (s5);
 
 float4 PS_main(VS_OUT input) : SV_Target
 {
-    return 5*textures[cbPerObject.info.textureAlbedo].Sample(point_Wrap, input.texCoord);
+    return textures[cbPerObject.info.textureAlbedo].Sample(point_Wrap, input.texCoord) * 5;
 }
```
Comment (fjz345) [inline StortSpelProjekt/Engine/src/Renderer/HLSL/SkyboxPixel.hlsl]: remove change

File: StortSpelProjekt/Engine/src/Renderer/HLSL/LightCalculations.hlsl [hunk 2]
```diff
@@ -25,7 +29,8 @@ float CalculateShadow(
 
 	// Check whether current fragPos is in shadow
 	float shadow = 0.0f;
-	float bias = 0.0003f;
+
+	float bias = 0.0f;//max(cbPerScene.maxBias * (1.0 - dot(normal, lightDir)), cbPerScene.minBias);
```
Comment (fjz345) [inline StortSpelProjekt/Engine/src/Renderer/HLSL/LightCalculations.hlsl]: Remove temp code

File: StortSpelProjekt/Game/src/main.cpp [hunk 3]
```diff
@@ -188,66 +188,66 @@ Scene* GetDemoScene(SceneManager* sm)
...
+    
     mc->SetModel(sphereModel);
     mc->SetDrawFlag(FLAG_DRAW::DRAW_OPAQUE | FLAG_DRAW::GIVE_SHADOW);
     tc->GetTransform()->SetScale(0.3f);
     tc->GetTransform()->SetPosition(-50.0f, 3.0f, 50.0f);
-
+    
     slc->SetColor({ 10.0f, 0.0f, 0.0f });
-    slc->SetAttenuation({ 1.0f, 0.027f, 0.0028f });
+    slc->SetAttenuation({ 1.0f, 0.14, 0.07f});
```
Comment (fjz345) [inline StortSpelProjekt/Game/src/main.cpp:208]: Sure this should be changed?

File: StortSpelProjekt/Game/src/main.cpp [hunk 4]
```diff
@@ -290,16 +290,16 @@ Scene* GetDemoScene(SceneManager* sm)
...
+    //{
+    //    zVal += 8;
+    //    entity = enH.AddExistingEnemy("enemy", float3{ xVal - 64, 1, zVal });
+    //    if ((i + 1) % 5 == 0)
+    //    {
+    //        xVal += 8;
+    //        zVal = 10;
+    //    }
+    //}
     /* ---------------------- Enemy -------------------------------- */
```
Comment (fjz345) [inline StortSpelProjekt/Game/src/main.cpp]: Remove changes before pull

File: StortSpelProjekt/Game/src/main.cpp [hunk 5]
```diff
@@ -328,7 +328,7 @@ void DemoUpdateScene(SceneManager* sm)
     ec->UpdateEmitter(L"Bruh");
 
     std::string name = "enemy";
-    for (int i = 1; i < 76; i++)
+    for (int i = 1; i < 1; i++)
```
Comment (fjz345) [inline StortSpelProjekt/Game/src/main.cpp]: remove before pull

**Review by fjz345 — CHANGES_REQUESTED** [review]
elloh

File: StortSpelProjekt/Engine/src/Renderer/HLSL/SkyboxPixel.hlsl (hunk 1 above)
Comment (jocke1995) [reply StortSpelProjekt/Engine/src/Renderer/HLSL/SkyboxPixel.hlsl]: yes

File: StortSpelProjekt/Engine/src/Renderer/HLSL/LightCalculations.hlsl (hunk 2 above)
Comment (jocke1995) [reply StortSpelProjekt/Engine/src/Renderer/HLSL/LightCalculations.hlsl]: yes

File: StortSpelProjekt/Game/src/main.cpp (hunk 3 above)
Comment (jocke1995) [reply StortSpelProjekt/Game/src/main.cpp:208]: yes

File: StortSpelProjekt/Game/src/main.cpp (hunk 4 above)
Comment (jocke1995) [reply StortSpelProjekt/Game/src/main.cpp]: yes

File: StortSpelProjekt/Game/src/main.cpp (hunk 5 above)
Comment (jocke1995) [reply StortSpelProjekt/Game/src/main.cpp]: yes

**Review by fjz345 — DISMISSED** [review]

**Review by fredrix96 — APPROVED** [review]
Well done. Nothing to comment about

**Review by fjz345 — APPROVED** [review]

**Pull request merged**